netty/src/main/java/io/grpc/transport/netty/NettyServerBuilder.java

```diff
@@ -48,6 +49,8 @@
  * A builder to help simplify the construction of a Netty-based GRPC server.
  */
 public final class NettyServerBuilder extends AbstractServerBuilder<NettyServerBuilder> {
+  public static final int DEFAULT_CONNECTION_WINDOW_SIZE = 1048576; // 1MiB
```

Comment:
I guess it makes sense to choose the same default for the server? Anyone remember why we are defaulting to 1MiB? @louiscryan @nmittler ?